FHEM/14_SD_UT.pm

```diff
@@ -338,1656 +340,1656 @@ use warnings;
 no warnings 'portable';  # Support for 64-bit ints required
 #use SetExtensions;
 
-#$| = 1;		#Puffern abschalten, Hilfreich für PEARL WARNINGS Search
+#$| = 1;  #Puffern abschalten, Hilfreich für PEARL WARNINGS Search
 
 sub SD_UT_bin2tristate($);
 sub SD_UT_tristate2bin($);
```

Comment:
Das kannst Du löschen, das brauchen wir nicht in Perl.

Reply:
Dann nehme ich dies raus.